app/models/family.rb

```diff
@@ -1,19 +1,21 @@
 class Family < ApplicationRecord
+  has_paper_trail
```

Comment:
@benreyn I added paper trail here too as it was fairly straightforward. Are you happy with this, or would you prefer it to just be on the 3 mentioned? Even further, if you wanted it on any other models it would be a very quick commit?

Reply:
This is great, I think I intended to have it here as well but just missed making the issue.